student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, name, parent_permission, classroom)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] As I mentioned earlier about the `default` parameter before, first try to make the changes in the `person` class and then come back to this `Student` class, you need to understand that `Student` inherits the property from `Person` so the name and `parent_permission` should be default parameter as per instructions and according to that you should pass the parameterized value to `super` class in order to maintain the `integrity` in your code.